src/components/CallView/LocalMediaControls.vue

```diff
@@ -0,0 +1,351 @@
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div class="nameIndicator">
+		<div id="muteWrapper">
+			<button
+				id="mute"
+				v-tooltip="audioButtonTooltip"
+				:class="audioButtonClass"
+				class="force-icon-white-in-call icon-shadow"
+				@click="toggleAudio" />
+			<span v-show="model.attributes.audioAvailable"
+				ref="volumeIndicator"
+				class="volume-indicator"
+				:style="{ 'height': currentVolumeIndicatorHeight + 'px' }" />
+		</div>
+		<button
+			id="hideVideo"
+			v-tooltip="videoButtonTooltip"
+			:class="videoButtonClass"
+			class="force-icon-white-in-call icon-shadow"
+			@click="toggleVideo" />
+		<button
+			v-if="!screenSharingButtonHidden"
+			id="screensharing-button"
+			v-tooltip="screenSharingButtonTooltip"
+			:class="screenSharingButtonClass"
+			class="app-navigation-entry-utils-menu-button force-icon-white-in-call icon-shadow"
+			@click="toggleScreenSharingMenu" />
+		<div id="screensharing-menu" :class="{ open: screenSharingMenuOpen }" class="app-navigation-entry-menu">
+			<ul>
+				<li v-if="!model.attributes.localScreen && splitScreenSharingMenu" id="share-screen-entry">
+					<button id="share-screen-button" @click="shareScreen">
+						<span class="icon-screen" />
+						<span>{{ t('spreed', 'Share whole screen') }}</span>
+					</button>
+				</li>
+				<li v-if="!model.attributes.localScreen && splitScreenSharingMenu" id="share-window-entry">
+					<button id="share-window-button" @click="shareWindow">
+						<span class="icon-share-window" />
+						<span>{{ t('spreed', 'Share a single window') }}</span>
+					</button>
+				</li>
+				<li v-if="model.attributes.localScreen" id="show-screen-entry">
+					<button id="show-screen-button" @click="showScreen">
+						<span class="icon-screen" />
+						<span>{{ t('spreed', 'Show your screen') }}</span>
+					</button>
+				</li>
+				<li v-if="model.attributes.localScreen" id="stop-screen-entry">
+					<button id="stop-screen-button" @click="stopScreen">
+						<span class="icon-screen-off" />
+						<span>{{ t('spreed', 'Stop screensharing') }}</span>
+					</button>
+				</li>
+			</ul>
+		</div>
+	</div>
+</template>
+
+<script>
+import { Tooltip } from '@nextcloud/vue'
```

Comment:
If you import the item directly, the resulting JS files should be smaller
```suggestion
import Tooltip from '@nextcloud/vue/dist/Directives/Tooltip'
```

Reply:
Thanks for the tip, I will do that.